crates/interpreter/Cargo.toml

```diff
@@ -23,12 +23,15 @@ lazy_static = "1.4.0"
 wast = "70.0.0"
 
 [features]
+default = ["threads"]
```

Comment:
This shouldn't be enabled by default. Instead you can add new lines to `test.sh`. If this is for another reason that is not easy to fix, then let's add a TODO to remove it from default, because otherwise we are only testing with threads enabled.

Reply:
Yeah this shouldn't be here, I only added it for local development as a hint to the language server.